Jenkinsfile

```diff
@@ -59,20 +62,58 @@ pipeline {
                                 source ./.cicd_bootstrap.sh
                                 source "${CICD_ROOT}/deploy_ephemeral_env.sh"
                                 source "${CICD_ROOT}/cji_smoke_test.sh"
```

Comment:
I think you want to only run `deploy_ephemeral_env.sh` if your goal is to deploy the dependencies only. Also, you need to set the "no remove namespace" part to avoid the teardown to delete the whole namespace (same thing as discussed in the vulnerability-engine PR)